microraft-hocon/build.gradle.kts

```diff
@@ -0,0 +1,103 @@
+plugins {
+    `java-library`
+    alias(libs.plugins.defaults)
+    alias(libs.plugins.metadata)
+    alias(libs.plugins.javadocLinks)
+    `maven-publish`
+    signing
+    alias(libs.plugins.mavenCentralPublishing)
+    alias(libs.plugins.spotbugs)
+    checkstyle
+}
+
+group = "io.microraft"
+version = "0.9-SNAPSHOT"
+description = ""
+
+metadata {
+    moduleName = "io.microraft.hocon"
+    readableName = "MicroRaft HOCON Config Parser"
+    license {
+        apache2()
+    }
+    organization {
+        name = "MicroRaft"
+        url = "https://microraft.io"
+    }
+    developers {
+        register("metanet") {
+            fullName = "Ensar Basri Kahveci"
+            email = "ebkahveci@gmail.com"
+        }
+        register("mdogan") {
+            fullName = "Mehmet Dogan"
+            email = "mehmet@dogan.io"
+        }
+    }
+    github {
+        org = "MicroRaft"
+        pages()
+        issues()
+    }
+}
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(11)
+    }
+    withJavadocJar()
+    withSourcesJar()
+}
+
+tasks.withType<Jar>().configureEach {
+    manifest.attributes(
+        "Implementation-Title" to project.name,
+        "Implementation-Vendor" to metadata.organization.provider.flatMap { it.name },
+        "Implementation-Version" to provider { project.version.toString() },
+    )
+}
+
+
+repositories {
+    mavenCentral()
+}
+
```

Comment:
```suggestion
```
We already have centralized dependency declarations in libs.versions.toml, let's also centralize the repository declarations (in this project it is anyhow only mavenCentral). To do so, add the following to settings.gradle.kts:
```
dependencyResolutionManagement {
    @Suppress("UnstableApiUsage")
    repositories {
        mavenCentral()
    }
}
```